DataDefinitions/StarSystem.cs

```diff
@@ -214,8 +214,9 @@ private HashSet<Material> MaterialsOnBodies(IEnumerable<Body> bodies)
             return result;
         }
 
-        // Not intended to be user facing - discoverable bodies as reported by a discovery scan "honk"
-        public int discoverableBodies = 0;
+        // Discoverable bodies as reported by a discovery scan "honk"
+        [JsonProperty("discoverableBodies")]
+        public int discoverablebodies;
```

Comment:
As discussed, consider changing (including all other instances) to `totalBodies` for clarity.

Reply:
changed to `totalbodies`